### PR TITLE
 Add Notification Email Action [PREMIUM-223]

### PR DESCRIPTION
**Pull request by websupporter**
## Description

_N/A_

## Code review notes

_N/A_

## QA notes

_N/A_

## Linked PRs

https://github.com/mailpoet/mailpoet-premium/pull/710

## Linked tickets

 [PREMIUM-223]

## After-merge notes

_N/A_

[PREMIUM-223]: https://mailpoet.atlassian.net/browse/PREMIUM-223?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: mailpoet/assets/js/src/automation/integrations/mailpoet/steps/notification_email/index.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  group: 'actions',
+  title: __('Send notification email', 'mailpoet'),
+  description: __(
+    'Receive a notification when a contact reaches a specific point in automation.',
+    'mailpoet',
+  ),
+  subtitle: () => <LockedBadge text={__('Premium', 'mailpoet')} />,
+  foreground: '#00A32A',
+  background: '#EDFAEF',
+  icon: () => <Icon />,
```
Comment (JanJakes) [inline mailpoet/assets/js/src/automation/integrations/mailpoet/steps/notification_email/index.tsx:25]: See https://github.com/mailpoet/mailpoet-premium/pull/710#discussion_r1165619714.

Comment (websupporter) [reply mailpoet/assets/js/src/automation/integrations/mailpoet/steps/notification_email/index.tsx:25]: [aa501ca](https://github.com/mailpoet/mailpoet/pull/4849/commits/aa501ca7699f18be7f582cc41243e5e317821016)

**Comment by websupporter** [comment]
Thank you @JanJakes

**Review by JanJakes — APPROVED** [review]
Looks great, thanks! 👏

**Comment by Aschepikov** [comment]
Hi @websupporter I can activate automation without the required fields, as I understand in the ticket we should make all fields as required but I see that required is just a from email address 
image
image

**Comment by Aschepikov** [comment]
Also, the validation message for the empty field is like there is something 
image
For example validations on the send email
image

**Comment by Aschepikov** [comment]
With all fields filled works ok, we need to make all fields required like in the ticket, without all fields automation from this step and to the end doesn't work correctly.

**Comment by websupporter** [comment]
@Aschepikov please check the premium PR. [74d0684](https://github.com/mailpoet/mailpoet-premium/pull/710/commits/74d0684754a33c1e967313178be6ba4b52292654) should solve the issue.

**Review by Aschepikov — APPROVED** [review]
GJ!

**Pull request merged**